vagrant/bootstrap-user.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# ARM GCC
+wget -q -O gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2 "https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2019q4/gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2?revision=108bd959-44bd-4619-9c19-26187abf5225&la=en&hash=E788CE92E5DFD64B2A8C246BBA91A249CB8E2D2D"
+tar xf gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2
+mkdir -p ~/.local
+mv gcc-arm-none-eabi-9-2019-q4-major/* ~/.local/
+
+# Python environment
+python3 -m pip install --user cryptography pyasn1 pyyaml jinja2 cbor
+
+# TrustedFirmware-M and dependencies
+mkdir tfm
+cd ~/tfm
+git clone https://git.trustedfirmware.org/TF-M/trusted-firmware-m.git
+git clone https://github.com/ARMmbed/mbed-crypto.git -b mbedcrypto-3.0.1
+git clone https://github.com/ARM-software/CMSIS_5.git -b 5.5.0
+cd ~/tfm/CMSIS_5
+wget -q https://github.com/ARM-software/CMSIS_5/releases/download/5.5.0/ARM.CMSIS.5.5.0.pack
+unzip -o ARM.CMSIS.5.5.0.pack
+cd ~/tfm/trusted-firmware-m
+git remote add ARMmbed https://github.com/ARMmbed/trusted-firmware-m.git
+git fetch --all
+
+# Mbed tools
+python3 -m pip install mbed-cli
```

Comment:
Getting this error
```vagrant@vagrant:~/mbed-os-tf-m-regression-tests$ python3 build_tfm.py -t GNUARM -m ARM_MUSCA_B1
Traceback (most recent call last):
  File "build_tfm.py", line 36, in <module>
    from tools.toolchains import TOOLCHAIN_CLASSES, TOOLCHAIN_PATHS
  File "/home/vagrant/mbed-os-tf-m-regression-tests/mbed-os/tools/toolchains/__init__.py", line 20, in <module>
    from . import mbed_toolchain, arm, gcc, iar
  File "/home/vagrant/mbed-os-tf-m-regression-tests/mbed-os/tools/toolchains/mbed_toolchain.py", line 49, in <module>
    from ..config import (ConfigException, RAM_ALL_MEMORIES, ROM_ALL_MEMORIES)
  File "/home/vagrant/mbed-os-tf-m-regression-tests/mbed-os/tools/config/__init__.py", line 28, in <module>
    from jsonschema import Draft4Validator, RefResolver
ModuleNotFoundError: No module named 'jsonschema'
```
Probably needs installation of these modules
```
python3 -m pip install cryptography pyasn1 pyyaml jinja2 cbor prettytable intelhex
python3 -m pip install pyelftools numpy jsonschema future mbed-cli mercurial
```

Reply:
After fixing the above by installing missing modules down the way, when executing
`mbed compile -t GCC_ARM -m ARM_MUSCA_B1`
get an error
```
vagrant@vagrant:~/mbed-os-tf-m-regression-tests$ mbed compile -t GCC_ARM -m ARM_MUSCA_B1
[mbed] Working path "/home/vagrant/mbed-os-tf-m-regression-tests" (program)
[mbed] WARNING: Could not find mbed program in current path "/home/vagrant/mbed-os-tf-m-regression-tests".
       You can fix this by calling "mbed new ." in the root of your program.
---
[mbed] WARNING: Missing Python modules were not auto-installed.
       The Mbed OS tools in this program require the following Python modules: mbed_cloud_sdk, mbed_ls, mbed_host_tests, mbed_greentea, manifest_tool, icetea, pycryptodome, pyusb, cmsis_pack_manager, psutil, click
       You can install all missing modules by running "pip install -r requirements.txt" in "/home/vagrant/mbed-os-tf-m-regression-tests/mbed-os"
       On Posix systems (Linux, etc) you might have to switch to superuser account or use "sudo"
---
Building project mbed-os-tf-m-regression-tests (ARM_MUSCA_B1, GCC_ARM)
Scan: mbed-os-tf-m-regression-tests
Configuration error: Bootloader not supported on this target. ROM start not found in targets.json.
Configuration error: Bootloader not supported on this target. RAM start not found in targets.json.
Link: mbed-os-tf-m-regression-tests
/home/vagrant/.local/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld: warning: ./test/lib/TOOLCHAIN_GCC_ARM/libtfm_non_secure_tests.a(non_secure_suites.o) uses 2-byte wchar_t yet the output is to use 4-byte wchar_t; use of wchar_t values across objects may fail
/home/vagrant/.local/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld: warning: ./test/lib/TOOLCHAIN_GCC_ARM/libtfm_non_secure_tests.a(test_framework_integ_test_helper.o) uses 2-byte wchar_t yet the output is to use 4-byte wchar_t; use of wchar_t values across objects may fail
/home/vagrant/.local/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld: warning: ./test/lib/TOOLCHAIN_GCC_ARM/libtfm_non_secure_tests.a(test_framework.o) uses 2-byte wchar_t yet the output is to use 4-byte wchar_t; use of wchar_t values across objects may fail
/home/vagrant/.local/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld: warning: ./test/lib/TOOLCHAIN_GCC_ARM/libtfm_non_secure_tests.a(test_framework_helpers.o) uses 2-byte wchar_t yet the output is to use 4-byte wchar_t; use of wchar_t values across objects may fail
/home/vagrant/.local/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld: BUILD/ARM_MUSCA_B1/GCC_ARM/main.o: in function `main':
/home/vagrant/mbed-os-tf-m-regression-tests/./main.cpp:37: undefined reference to `TIMER1_Handler'
collect2: error: ld returned 1 exit status
[ERROR] /home/vagrant/.local/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld: warning: ./test/lib/TOOLCHAIN_GCC_ARM/libtfm_non_secure_tests.a(non_secure_suites.o) uses 2-byte wchar_t yet the output is to use 4-byte wchar_t; use of wchar_t values across objects may fail
/home/vagrant/.local/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld: warning: ./test/lib/TOOLCHAIN_GCC_ARM/libtfm_non_secure_tests.a(test_framework_integ_test_helper.o) uses 2-byte wchar_t yet the output is to use 4-byte wchar_t; use of wchar_t values across objects may fail
/home/vagrant/.local/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld: warning: ./test/lib/TOOLCHAIN_GCC_ARM/libtfm_non_secure_tests.a(test_framework.o) uses 2-byte wchar_t yet the output is to use 4-byte wchar_t; use of wchar_t values across objects may fail
/home/vagrant/.local/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld: warning: ./test/lib/TOOLCHAIN_GCC_ARM/libtfm_non_secure_tests.a(test_framework_helpers.o) uses 2-byte wchar_t yet the output is to use 4-byte wchar_t; use of wchar_t values across objects may fail
/home/vagrant/.local/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld: BUILD/ARM_MUSCA_B1/GCC_ARM/main.o: in function `main':
/home/vagrant/mbed-os-tf-m-regression-tests/./main.cpp:37: undefined reference to `TIMER1_Handler'
collect2: error: ld returned 1 exit status
```

Reply:
Issue with using `-c ConfigRegressionIPC.cmake`